packages/babel-plugin-transform-function-name/test/fixtures/function-name/unicode-id-not-supported/options.json

```diff
@@ -0,0 +1,4 @@
+{
+    "plugins": ["transform-unicode-escapes"],
+    "targets": { "firefox": 60 }
```

Comment:
This test case is testing the behaviour of `transform-function-name` when unicode id is _not_ supported, however, `Firefox 60` does support unicode escapes. In fact it has supported since 53. So we should specify `Firefox 52` or other versions here.

Reply:
Okay, I will use `Firefox 52` for not supported unicode escapes.

Reply:
After changing the targets it still gives the same result for different targets, I don't know what's happening?

Reply:
`supportUnicodeId` is same for all the tests.